client/src/components/OtherComponents/NavBar.js

```diff
@@ -7,25 +7,41 @@ const NavBar = () => {
       <Nav>
         <Bars />
         <NavMenu>
-          <NavLink exact to="/" activeStyle={{fontWeight: "bold"}}>
+          <NavLink exact to="/" activeStyle={{ fontWeight: "bold" }}>
             TabSplitter
           </NavLink>
-          
-          <NavLink exact to="/contacts" activeStyle={{fontWeight: "bold"}}>
+
+          <NavLink exact to="/contacts" activeStyle={{ fontWeight: "bold" }}>
```

Comment:
❓ can you use CSS here?

Reply:
The CSS for the NavBar.js is mainly handled in NavbarElements.js. Moved this to that sheet.